charts/airflow/values.yaml

```diff
@@ -328,6 +328,12 @@ airflow:
     ##
     extraVolumes: []
 
+    ## extra containers for the pod template
+    ## - spec for Volume:
+    ##   https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.20/#container-v1-core
+    ##
+    extraContainers: []
```

Comment:
nit: I like to keep the `extraVolumeMounts` and `extraVolumes` at the bottom, can you please move `extraContainers` directly above `extraVolumeMounts`?

Reply:
corrected